.github/CODEOWNERS

```diff
@@ -29,6 +29,7 @@
 # Emiter Shared
 ######################
 /eng/emitters/                               @m-nash @iscai-msft
+/docs/samples/client/                        @lmazuel @JoshLove-msft @iscai-msft @srnagar @anuchandy @weidongxu-microsoft @haolingdong-msft @XiaofeiCao @tadelesh @msyyc @joheredi @timotheeguerin @qiaozha @MaryGao
```

Comment:
We don't put docs in the docs directory anymore, they are here: website/src/content/docs/docs

But is this meant to go in the samples package, or elsewhere?

Reply:
We had a long discussion, and concluded on that folder:
https://teams.microsoft.com/l/message/19:16f8f1ed70d044139be153707bdaee0e@thread.skype/1741038119806?tenantId=72f988bf-86f1-41af-91ab-2d7cd011db47&groupId=3e17dcb0-4257-4a30-b843-77f47f1d4121&parentMessageId=1741038119806&teamName=Azure%20SDK&channelName=TypeSpec%20Engineering&createdTime=1741038119806